### PR TITLE
Completed Assignment 3

### PR DESCRIPTION
**Pull request by JorgeOP46n2**

### CODE REVIEW

**Comment by ocorcho** [comment]
Assignment 3 accepted. Now @dgarijo can chheck assignment 4

File: Assignment4/JorgeOP46n2-a180113/task06.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1NPTMTAkCR_p4HWRPCu15gIzLBYzRdjhX
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/JorgeOP46n2-a180113/task06.py:12]: won't work in python. This is notebook magic

File: Assignment4/JorgeOP46n2-a180113/task06.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+for s, p, o in g.triples((ns.JaneSmith, None, None)):
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, vcard.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/JorgeOP46n2-a180113/task06.py:66]: vcard.work does not exist. Use your own property

File: Assignment4/JorgeOP46n2-a180113/task07.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+
+print("SPARQL")
+q1 = prepareQuery("""
+    SELECT DISTINCT ?persona ?propiedad WHERE{
+    {?persona rdf:type ns:Person.}
+    
+    UNION{
+      ?subclase rdfs:subClassOf ns:Person.
+      ?persona rdf:type ?subclase
```
Comment (dgarijo) [inline Assignment4/JorgeOP46n2-a180113/task07.py:96]: for this one and previous one, it's better to use subclassOf*, as you would cover the full hierarcy.

**Review by dgarijo — CHANGES_REQUESTED** [review]
Small changes requested

**Pull request merged**